### PR TITLE
Env Var Refactor

### PR DESCRIPTION
**Pull request by mairh**
- Shall we go with this?

### CODE REVIEW

File: .spinrc.json [hunk 1]
```diff
@@ -11,11 +11,11 @@
     },
     "android": {
       "stack": ["react-native", "android"],
-      "enabled": false
+      "enabled": true
```
Comment (vlasenko) [inline .spinrc.json]: This file shouldn't be changed

Comment (mairh) [reply .spinrc.json]: I think you commented on the previous version before I pushed the update

**Comment by vlasenko** [comment]
@mairh Looks good now, thank you Ujjwal!

**Comment by mairh** [comment]
@mitjade Anything you wanna add?

**Comment by mitjade** [comment]
@mairh All good!

**Pull request merged**